app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/JourneyUserType.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cgtpropertydisposalsfrontend.models
+
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.ids.{CgtReference, GGCredId, NINO, SAUTR}
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.onboarding.email.Email
+
+sealed trait JourneyUserType extends Product with Serializable
```

Comment:
this case class is to describe the user type based on the data retrieved from auth - maybe something like RetrievedUserType would be more accurate?

Reply:
yep, that name reflects better the semantics - I will change it.

Reply:
fixed